commons/src/main/scala/dagr/commons/CommonsDef.scala

```diff
@@ -62,4 +63,35 @@ object DagrDef {
 
   /** The type of identifier used to uniquely identify tasks tracked by the execution system. */
   type TaskId = BigInt
+
```

Comment:
I understand - but I still feel weird about it.  My thought was that commons was to be generally re-usable scala utilities, not just a place to deposit anything we wanted to use across two projects.